models/sources/src_accounts.yml

```diff
@@ -181,10 +181,3 @@ sources:
               - incremental_not_null:
                   date_column_name: "batch_run_date"
                   greater_than_equal_to: "2 day"
-
-          - name: batch_insert_ts
```

Comment:
I think source and staging files should keep the `batch_insert_ts` fields to be in sync with the columns from the raw base tables from `crypto-stellar.crypto_stellar` dataset

Reply:
Yes, added it back in https://github.com/stellar/stellar-dbt-public/pull/80/commits/1819e31dc1182190e3f195fd66795e09ff9ca477